packages/ckeditor5-mention/src/mentionediting.ts

```diff
@@ -62,13 +75,16 @@ export default class MentionEditing extends Plugin {
 
 		doc.registerPostFixer( writer => removePartialMentionPostFixer( writer, doc, model.schema ) );
 		doc.registerPostFixer( writer => extendAttributeOnMentionPostFixer( writer, doc ) );
-		doc.registerPostFixer( writer => selectionMentionAttributePostFixer( writer, doc ) );
+		doc.registerPostFixer( writer => selectionMentionAttributePostFixer( writer, doc ) as boolean );
```

Comment:
Instead of cast here, just add `return false` at the end of `selectionMentionAttributePostFixer`.